ui/src/app/cohort-search/modifier-page/modifier-page.component.html

```diff
@@ -100,11 +100,11 @@
 <div class="container count-preview">
   <div class="row">
     <div class="col-lg-3">
-      <button type="button" [ngClass]="[ preview.get('requesting') || showError? 'greyed-button': 'calculate-btn']"
-        class="btn "
+      <button type="button"
+        class="btn calculate-btn"
         [clrLoading]="preview.get('requesting')"
         (click)="requestPreview()"
-        [disabled] = "!formChanges">
+        [disabled]="disableCalculate">
         Calculate
       </button>
     </div>
```

Comment:
Also, we need to limit the integers that can be entered in the age at event field and number of occurrences as well. I was able to enter 100000000000000000000000000000000000000